apps/system/test/unit/app_install_manager_test.js

```diff
@@ -339,11 +339,8 @@ suite('system/AppInstallManager >', function() {
       });
 
       test('the dialog should be hidden after press home', function() {
-        var evt = {
-          type: 'home'
-        };
         assert.equal('visible', AppInstallManager.dialog.className);
-        AppInstallManager.handleHomeButtonPressed(evt);
+        AppInstallManager.cancelInstallation();
```

Comment:
With this approach we're not covering with tests that the event listeners are properly set.
Instead of calling `cancelInstallation()` directly you can dispatch real event ie.
```
        window.dispatchEvent(new CustomEvent('holdhome'));
```

This will make the test more external and test more code for the same amount of test lines :)